ui/app/AppLayouts/Chat/panels/communities/CommunityOverviewSettingsPanel.qml

```diff
@@ -8,10 +8,19 @@ import StatusQ.Controls 0.1
 import StatusQ.Components 0.1
 
 import "../../layouts"
+import "../../popups/community"
+
+import utils 1.0
 
 StackLayout {
     id: root
 
+    property bool hasAddedContacts: false
+    property var activeCommunity
+    property var communitySectionModule
+    property var store
```

Comment:
According to QML core architecture panels should not have any dependency to a store or a view.
https://doc-snapshots.qt.io/neptune3ui/neptune3ui-application-architecture.html
https://hackmd.io/@status-desktop/main/https%3A%2F%2Fhackmd.io%2F%40status-desktop%2FB1eOaf-nd

The solution would be to propagate all banner signals to `CommunitySettingsView`. 
The flow should look as follows `CommunityOverviewSettingsPanel`-signals->`CommunitySettingsView`-calls->`store`-triggers->`popups`